src/Microsoft.OData.Core/ODataBatchReader.cs

```diff
@@ -36,6 +30,9 @@ public sealed class ODataBatchReader : IODataBatchOperationListener
         /// <summary>The dependency injection container to get related services.</summary>
         private readonly IServiceProvider container;
 
+        /// <summary>The input context to read the content from.</summary>
+        private ODataInputContext inputContext;

```

Comment:
>private ODataInputContext inputContext; [](start = 8, length = 39)

should this be readonly?

Reply:
This private field only have a getter, therefore it is equivalent to readonly.